__tests__/wikiQuery.js

```diff
@@ -0,0 +1,45 @@
+import { getTeamList } from '../app/utils/wikiQuery'
+
+describe('getData', () => {
+    it('should run', () => {
+        // Arrange
+        const firstContestantsFirstName = "Some"
+        const secondContestantsFirstName = "SomeGuys"
+
+        const listOfContestants = [
+            {
+                name: firstContestantsFirstName + " Guy",
+                status: "Participating"
+            },
+            {
+                name: secondContestantsFirstName + " Brother",
+                status: "Participating"
+            }]
+
+        // Act
+        var result = getTeamList(listOfContestants)
+
+        // Assert
+        expect(result).not.toBeNull()
```

Comment:
I would go so far to say that we're not just expecting a non null result here, but also there to be some specific values regarding some guy and his brother. 

Although, I believe it not being null is certainly sufficient for now.

Reply:
I totally agree, I do have other tests coming soon, but I also wanted to get this in ASAP so that in case others start making changes they can use this as a guide for how to add tests to make sure their changes are working!